lib/_stream_duplex.js

```diff
@@ -98,6 +98,13 @@ Object.defineProperty(Duplex.prototype, 'writableLength', {
   }
 });
 
+Object.defineProperty(Duplex.prototype, 'writableFinished', {
+  enumerable: false,
```

Comment:
the default of `enumerable` is false, this is unnecessary

see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty

Reply:
Not opposed to removing it, but certainly OK to leave it. There's nothing wrong with including it for extra clarity when reading the code. Readability and maintainability are more important than being as concise as possible.

Reply:
Hmmm...although I guess if you're depending on the defaults for all the other properties, including just the one might be a little strange? Anyway, I'm fine with whatever personally.

Reply:
It's consistent with the other getters definitions, for example https://github.com/nodejs/node/blob/187b05640837443353ec4183e203f352ff0af88a/lib/_stream_writable.js#L597-L605

but again it should be on `Writable.prototype` not `Duplex.prototype.`

Reply:
Actually it should be on both, See comment below.